file_store/src/cli/dump.rs

```diff
@@ -163,8 +164,14 @@ impl Cmd {
                     let packet_report = PacketRouterPacketReportV1::decode(msg)?;
                     print_json(&json!({
                         "oui": packet_report.oui,
-                        "timestamp": packet_report.gateway_timestamp_ms,
-
+                        "timestamp": packet_report.gateway_timestamp_ms}))?;
+                }
+                FileType::PriceReport => {
+                    let manifest = PriceReportV1::decode(msg)?;
+                    print_json(&json!({
+                        "price": manifest.price,
+                        "timestamp": manifest.timestamp,
+                        "token_type": BlockchainTokenTypeV1::from_i32(manifest.token_type),
```

Comment:
is `from_i32` an infallible function call? i thought those all had to be result types

Reply:
I think `manifest.token_type()` will return the enum.

Reply:
Oh yeah; I keep forgetting about those functions

Reply:
Yup, ty. Updated [f8ab0b3](https://github.com/helium/oracles/pull/400/commits/f8ab0b3f81d8ddf7315cdd800beaa15b769ab9af)